cdap-docs/admin-manual/source/troubleshooting.rst

```diff
@@ -2,13 +2,29 @@
     :author: Cask Data, Inc.
     :copyright: Copyright © 2014 Cask Data, Inc.
 
-============================================
-Troubleshooting CDAP
-============================================
+.. _admin-troubleshooting:
```

Comment:
Added note about developers' manual to https://issues.cask.co/browse/CDAP-3278. Will move other material to the FAQs.